charts/tempo-distributed/templates/gateway/deployment-gateway.yaml

```
@@ -1,10 +1,11 @@
 {{- if .Values.gateway.enabled }}
```

shouldn't this be
```suggestion
{{- if and .Values.gateway.enabled (not .Values.enterprise.enabled) }}
```
Because now it seems like you run both gateways when enterprise is enabled

---

I don't suppose a user would want to run both, but since both gateways default to false, I thought just switching in the specific component for each one reduced the logic.  Users who want to run only one gateway will need to enable only one gateway in the config.

Alternatively, this could be the kind of thing we put in a validation template, but is not implemented in this pr.